src/core_data_transform.sql

```diff
@@ -0,0 +1,137 @@
+-- This assumes that the original data has been imported into the database
+-- and that the new tables have been created based on the schema in the
+-- B2_build_movie_db.sql file. The following queries will transform the
+-- data from the original tables into the new tables.
+
+CREATE SCHEMA IF NOT EXISTS original;
+
+DO $$
+BEGIN
+    IF EXISTS (SELECT 1 FROM pg_tables WHERE schemaname = 'public' AND tablename = 'title_basics') THEN
+        ALTER TABLE public.title_basics SET SCHEMA original;
+    END IF;
+
+    IF EXISTS (SELECT 1 FROM pg_tables WHERE schemaname = 'public' AND tablename = 'title_ratings') THEN
+        ALTER TABLE public.title_ratings SET SCHEMA original;
+    END IF;
+
+    IF EXISTS (SELECT 1 FROM pg_tables WHERE schemaname = 'public' AND tablename = 'title_crew') THEN
+        ALTER TABLE public.title_crew SET SCHEMA original;
+    END IF;
+
+    IF EXISTS (SELECT 1 FROM pg_tables WHERE schemaname = 'public' AND tablename = 'title_principals') THEN
+        ALTER TABLE public.title_principals SET SCHEMA original;
+    END IF;
+
+    IF EXISTS (SELECT 1 FROM pg_tables WHERE schemaname = 'public' AND tablename = 'title_akas') THEN
+        ALTER TABLE public.title_akas SET SCHEMA original;
+    END IF;
+
+    IF EXISTS (SELECT 1 FROM pg_tables WHERE schemaname = 'public' AND tablename = 'title_episode') THEN
+        ALTER TABLE public.title_episode SET SCHEMA original;
+    END IF;
+
+    IF EXISTS (SELECT 1 FROM pg_tables WHERE schemaname = 'public' AND tablename = 'name_basics') THEN
+        ALTER TABLE public.name_basics SET SCHEMA original;
+    END IF;
+
+    IF EXISTS (SELECT 1 FROM pg_tables WHERE schemaname = 'public' AND tablename = 'omdb_data') THEN
+        ALTER TABLE public.omdb_data SET SCHEMA original;
+    END IF;
+
+    IF EXISTS (SELECT 1 FROM pg_tables WHERE schemaname = 'public' AND tablename = 'wi') THEN
+        ALTER TABLE public.wi SET SCHEMA original;
+    END IF;
+END $$;
+
+-- Countries
+
+INSERT INTO country (code, name) 
+SELECT DISTINCT region, '' FROM original.title_akas;
+
+-- Languages
+
+INSERT INTO "language" (code, name)
```

Comment:
<img width="347" alt="image" src="https://github.com/user-attachments/assets/c458c241-3728-48f7-b9a6-e2886747de3c">

This probably needs a fix

Reply:
yep, should be fixed now